cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,143 @@ Cask Data Application Platform Release Notes
+  old data in the old format.
+
+- `CDAP-797 <https://issues.cask.co/browse/CDAP-797>`__ - 
+  When running secure Hadoop clusters, debug logs from MapReduce programs are not
+  available.
+
+- `CDAP-1007 <https://issues.cask.co/browse/CDAP-1007>`__ -
+  If the Hive Metastore is restarted while the CDAP Explore Service is running, the
+  Explore Service remains alive, but becomes unusable. To correct, restart the CDAP Master
+  |---| which will restart all services |---| as described under "Starting CDAP Services"
+  for your particular Hadoop distribution in the :ref:`Installation documentation <installation-index>`.
+
+- `CDAP-1587 <https://issues.cask.co/browse/CDAP-1587>`__ - 
+  User datasets with names starting with ``"system"`` can potentially cause conflicts.
+
+- `CDAP-2632 <https://issues.cask.co/browse/CDAP-2632>`__ -
+  The application in the `cdap-kafka-ingest-guide 
+  <https://github.com/cdap-guides/cdap-kafka-ingest-guide/tree/release/cdap-3.0-compatible>`__ 
+  does not run on Ubuntu 14.x as of CDAP 3.0.x.
+
+- `CDAP-2721 <https://issues.cask.co/browse/CDAP-2721>`__ -
+  Metrics for :ref:`TimePartitionedFileSets <datasets-timepartitioned-fileset>` can show 
+  zero values even if there is data present.
+  
+- `CDAP-2831 <https://issues.cask.co/browse/CDAP-2831>`__ -
+  A workflow that is scheduled by time will not be run between the failure of the primary
+  master and the time that the secondary takes over. This scheduled run will not be
+  triggered at all. There is no warnings or messages about the missed run of the
+  workflow.
+
+- `CDAP-2945 <https://issues.cask.co/browse/CDAP-2945>`__ -
+  There is a problem in the PartitionedFileSet causing MapReduce to fail if the input
+  partition filter does not match any partitions.
+
+- `CDAP-3000 <https://issues.cask.co/browse/CDAP-3000>`__ -
+  There is a problem with the Workflow token being in an inconsistent state for nodes in a
+  fork while the fork is still running. It becomes consistent after the join.
+
+- `CDAP-3221 <https://issues.cask.co/browse/CDAP-3221>`__ -
+  There is a problem when running Standalone mode: if an adapter is configured incorrectly
+  such that it leads to a MapReduce job to fail repeatedly, then the SDK hits an OOM exception due to ``perm gen``.
+  The Standalone needs restarting at this point.
+
+- `CDAP-3262 <https://issues.cask.co/browse/CDAP-3262>`__ -
+  There is a problem under Microsoft Windows when using the CDAP Standalone scripts with a  
+  JAVA_HOME that is defined as a path with spaces in it. A workaround is to use a definition of 
+  JAVA_HOME that does not include spaces.
+
+- `CDAP-3492 <https://issues.cask.co/browse/CDAP-3492>`__ -
+  In the CDAP CLI, executing ``'select *'`` from a dataset with many fields generates an error.
+  
+- `CDAP-3641 <https://issues.cask.co/browse/CDAP-3641>`__ -
+  ``WorkflowStatsSLAHttpHandler`` hangs if units are not provided.
```

Comment:
This is too low level. Users who consume this don't know about SLAHttpHandler. What end point will cause this issue